src/Modules/ModParser.lua

```diff
@@ -1540,6 +1545,9 @@ local specialModList = {
 		mod("EnemyModifier", "LIST", { mod = mod("PhysicalDamageTaken", "INC", num) }),
 		mod("EnemyModifier", "LIST", { mod = mod("FireDamageTaken", "INC", num) }),
 	} end,
+	["enemies near your totems deal (%d+)%% less damage"] = function(num) return {
```

Comment:
This line can probably go in the `preFlagList` as "enemies near your totems deal".

Reply:
moved